pkg/component/istio/charts/istio/istio-istiod/templates/destinationrule.yaml

```diff
@@ -1,12 +0,0 @@
-apiVersion: networking.istio.io/v1beta1
-kind: DestinationRule
-metadata:
-  name: default
-  namespace: {{ .Release.Namespace }}
-spec:
-  host: "*"
-  exportTo:
-  - "*"
-  trafficPolicy:
-    tls:
-      mode: ISTIO_MUTUAL
```

Comment:
What does the removal of the default destination rule imply? My assumption is that this is the reason why it was always required to specify a destination rule as otherwise this destination rule would have enabled mutual authentication.

Reply:
Yes, it seems that after the removal the other destination rules are not needed anymore.

Reply:
The other destination rules are no longer needed for switching off mutual TLS, but the contain other important configuration, most notably the locality settings.

I am not sure whether we want to keep this default rule as a safety net or if this is just obscure security.

WDYT?

Reply:
I added the destination rule again.  We can think about it in a future PR. Maybe, we can set the locality setting globally with an empty TLS config here.